### ISSUE TITLE
Feature Request: Non-case sensitive name when tagging user in a card

### ISSUE THREAD

**Issue by Usernameisalreadytaken99** [wekan/wekan#1938]
Hey all,

It would be useful for the name suggestions when @mentioning users to NOT be case sensitive. 

At the moment if I try to assign myself or another user a card I type in bla bla bla @**n**ameofuser and it doesn't work. Instead I have to write bla bla bla @**N**ameOfUser with the correct case.

Thanks,
